### PR TITLE
English homework has been done again!

### PR DESCRIPTION
**Pull request by Svetlana101010**
Corrected homework after understanding video.

### CODE REVIEW

File: homework/english/svet_konduhowich/lecture1/task1.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+10. Is his wife having coffee at the moment?
+11. Let’s go for a walk. It isn't raining.
+12. Look! The boys are coming home.
+13. Does Maila like Chinese food?
+14. My grandmother doesn't work. She is on pension.
+15. What is your brother watching now?
+16. Where do you live?
+17. Do you play the piano well?
+18. Where are their brothers swimming now?
+19. What do them eating at breakfast?
```
Comment (VolhaZh) [inline homework/english/svet_konduhowich/lecture1/task1.md:20]: they

**Review by VolhaZh — CHANGES_REQUESTED** [review]

**Review by Alex99905 — APPROVED** [review]

**Review by Denis-Pikus — APPROVED** [review]
lgfm

**Review by Alexandr-R-94 — APPROVED** [review]
Lgmt

File: homework/english/svet_konduhowich/lecture1/task1.md (hunk 1 above)
Comment (Svetlana101010) [reply homework/english/svet_konduhowich/lecture1/task1.md:20]: I think it is their i right

**Review by i3oi3ka — DISMISSED** [review]
lgfm

File: homework/english/svet_konduhowich/lecture1/task1.md (hunk 1 above)
Comment (VolhaZh) [reply homework/english/svet_konduhowich/lecture1/task1.md:20]: it seems must be a subject(i, you, he, she, it, we, they)
they do

**Review by Alex99905 — DISMISSED** [review]

**Comment by puhlikov** [comment]
lgfm

**Comment by puhlikov** [comment]
lgfm

**Review by Svetlana101010 — COMMENTED** [review]
i have changed their for they at 18

**Review by puhlikov — APPROVED** [review]

**Review by VolhaZh — APPROVED** [review]

**Review by Alex99905 — APPROVED** [review]

**Review by anastasinastena — APPROVED** [review]

**Pull request merged**